openassessment/xblock/grade_mixin.py

```diff
@@ -195,6 +198,27 @@ def _is_incomplete(step):
             {'incomplete_steps': incomplete_steps}
         )
 
+    def render_grade_override(self, override_score):
```

Comment:
Agree but following the existing convention (3 existing similar methods) makes it easy to see it does the same thing.